src/directives/popover.js

```diff
@@ -0,0 +1,147 @@
+goog.provide('ngeo.popoverDirective');
+goog.provide('ngeo.popoverAnchorDirective');
+goog.provide('ngeo.popoverContentDirective');
+goog.provide('ngeo.PopoverController');
+
+goog.require('ngeo');
+
+/**
+ * Provides a directive used to display a Bootstrap popover.
+ *
+ *<div ngeo-popover>
+ *  <a ngeo-popover-anchor class="btn btn-info">anchor 1</a>
+ *  <div ngeo-popover-content="">
+ *    <ul>
+ *      <li>action 1:
+ *        <input type="range"/>
+ *      </li>
+ *    </ul>
+ *  </div>
+ *</div>
+ * @ngdoc directive
+ * @ngInject
+ * @ngname ngeoPopover
+ * @return {angular.Directive} The Directive Definition Object.
+ */
+ngeo.popoverDirective = function() {
+  return {
+    restrict: 'A',
+    scope : true,
+    controller: 'NgeoPopoverController',
+    link : function(scope, elem, attrs, controller) {
+      var ngeoPopoverCtrl = controller;
+
+      ngeoPopoverCtrl.anchorElm .on('hidden.bs.popover', function() {
+        /**
+         * @type {{inState : Object}}
+         */
+        var popover = ngeoPopoverCtrl.anchorElm.data('bs.popover');
+        popover['inState'].click = false;
+      });
+
+      ngeoPopoverCtrl.anchorElm.on('inserted.bs.popover', function() {
+        ngeoPopoverCtrl.shown = true;
+        ngeoPopoverCtrl.bodyElm.parent().on('click', function(e) {
+          e.stopPropagation();
+        })
+      });
+
+      ngeoPopoverCtrl.anchorElm.popover({
+        container: 'body',
+        html: true,
+        content: ngeoPopoverCtrl.bodyElm
+      });
+
+      scope.$on('$destroy', function() {
+        ngeoPopoverCtrl.anchorElm .unbind('inserted.bs.popover');
+        ngeoPopoverCtrl.anchorElm .unbind('hidden.bs.popover');
```

Comment:
same here: space between `anchorElm` and ` .unbind`

I am surprised that the linter does not catch this.